networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/gossip/config/GossipScoringConfig.java

```diff
@@ -163,5 +167,26 @@ public Builder opportunisticGraftThreshold(final Double opportunisticGraftThresh
       this.opportunisticGraftThreshold = opportunisticGraftThreshold;
       return this;
     }
+
+    public Builder directPeers(final List<String> directPeers) {
+      checkNotNull(directPeers);
+
+      class GossipScoringDirectPeerManager implements DirectPeerManager {
+        private final List<String> directPeers;
+
+        @Override
+        public boolean isDirectPeer(NodeId peerId) {
+          return directPeers.contains(peerId.toString());
+        }
+
+        public GossipScoringDirectPeerManager(List<String> directPeers) {
+          this.directPeers = directPeers;
+        }
+      }
```

Comment:
I'm not sure whether adding an anonymous class here is good or not from a coding style standpoint. I just want something simple that implements the `DirectPeerManager` interface. Maybe the Teku team will help me